data/net/minecraft/world/level/levelgen/flat/FlatLevelGeneratorPresets.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/world/level/levelgen/flat/FlatLevelGeneratorPresets
+	METHOD bootstrap (Lnet/minecraft/core/Registry;)Lnet/minecraft/core/Holder;
+		ARG 0 registry
+	METHOD register (Ljava/lang/String;)Lnet/minecraft/resources/ResourceKey;
+		ARG 0 name
+	CLASS Bootstrap
+		METHOD <init> (Lnet/minecraft/core/Registry;)V
+			ARG 1 presets
+		METHOD register (Lnet/minecraft/resources/ResourceKey;Lnet/minecraft/world/level/ItemLike;Lnet/minecraft/resources/ResourceKey;Ljava/util/Set;ZZ[Lnet/minecraft/world/level/levelgen/flat/FlatLayerInfo;)Lnet/minecraft/core/Holder;
+			ARG 1 presets
+			ARG 2 itemLike
```

Comment:
This parameter is transformed into and passed as the `displayItem` record component of `FlatLevelGeneratorPreset`.
```suggestion
			ARG 2 displayItem
```